systers_portal/meetup/tests/test_views.py

```diff
@@ -239,6 +239,13 @@ def setUp(self):
                                              meetup_location=self.meetup_location,
                                              created_by=self.systers_user,
                                              last_updated=timezone.now())
+        self.meetup3 = Meetup.objects.create(title='Foo Baz', slug='foobar',
```

Comment:
There must have been a reason to add another meetup object to the tests, but I can't find any.
Anything you forgot to include?